vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -10730,6 +10748,7 @@ quantitykind:Momentum
   a qudt:QuantityKind ;
   qudt:applicableUnit unit:KiloGM_F-M-PER-SEC ;
   qudt:dbpediaMatch "http://dbpedia.org/resource/Momentum"^^xsd:anyURI ;
+  qudt:hasDimensionVector qkdv:A0E0L1I0M1H0T-1D0 ;
```

Comment:
The dimension is correct, but you have uncovered an error in the referring unit. I will fix the error in unit:KiloGM_F-M-PER-SEC, which should not be referring to this quantity kind.